lib/v1/config.js

```diff
@@ -1,7 +1,12 @@
 const KEYS_URL = 'https://keys.evervault.com';
-const METRICS_URL = ' https://metrics.evervault.com';
+const METRICS_URL = 'https://metrics.evervault.com';
 
-module.exports = (teamId) => ({
+const DEFAULT_CONFIG_URLS = {
+	keysUrl: KEYS_URL,
+  metricsUrl: METRICS_URL
+};
+
+module.exports = (teamId, configUrls = DEFAULT_CONFIG_URLS) => ({
```

Comment:
Rather than returning the object straight away, you should add some checks that `configUrls` contains the expected fields/types and fall back to default values if not